runtests/settings.py

```diff
@@ -101,6 +101,8 @@
 
 ROOT_URLCONF = 'urls'
 
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```

Comment:
should we use default bigAutoField or add per app https://github.com/celery/django-celery-results/blob/master/django_celery_results/apps.py#L15 default_auto_field = 'django.db.models.AutoField' to keep backward compat?

Reply:
that's in runtests. do you mean adding the option to `actstream.apps.ActstreamConfig` like celery did?

Reply:
yes that is what I meant